apps/asap-server/test/utils/sync-google-calendar.test.ts

```diff
@@ -17,6 +17,16 @@ jest.mock('googleapis', () => ({
   },
 }));
 
+jest.mock('luxon', () => ({
+  DateTime: {
+    utc: () => ({
+      plus: () => ({
+        toISO: () => 'six-months-from-now',
```

Comment:
I would probably stick to mocking the Date.now constructor, like in some other files. What you do here is you mock the method at the end of the method chain, which is wrong for two reasons:
1) You don't really allow for any refactoring, if somebody want's to use a different way to get that date they have to change the mocks
2) You are not actually testing a very important logic - we don't know if the period was added correctly. I would probably pick a date, make Date.now return it, calculate the +6 months myself and check that the calculated ISO is in line with it

Reply:
Agreed. I will change it